### PR TITLE
Integrate the terraform workflow into the gitlab CI

### PR DESCRIPTION
**Pull request by genalvdom**
# Pull Request Template

Thanks you for your contribution to the `RSS Feed Generator` repo. 

## Description

* Integrate the terraform workflow into the gitlab CI

## Before creating the pull  request

-  My code follows the contributing guidelines of this project
- I have performed a self-review of my own code
- I have made corresponding changes to the documentation

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+prepare:
+  image: alpine
+  stage: prepare
+  script:
+    - echo -n $service_account > credentials.json
+  artifacts:
+    paths:
+      - credentials.json
+  tags:
+    - test2
```
Comment (sjortiz) [inline .gitlab-ci.yml]: Investigate the variable type file, it might help removing this step

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+stages:
+  - prepare
+  - init
+  - deploy
```
Comment (sjortiz) [inline .gitlab-ci.yml:2]: the deploy stage doesn't exist

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    name: stashconsulting/terraform-docker:gcloud-latest
+    entrypoint: [""]
+  stage: init
+  script:
+    - gcloud auth activate-service-account --key-file credentials.json
+    - gcloud auth configure-docker
+    - terraform init
+    - terraform apply -auto-approve -var image_id=$image -var project=$project
+  tags:
+    - test2
```
Comment (sjortiz) [inline .gitlab-ci.yml]: Use a descriptive tag

File: .gitlab-ci.yml (hunk 1 above)
Comment (sjortiz) [inline .gitlab-ci.yml]: use a descriptive tag

**Review by sjortiz — APPROVED** [review]
some comments

File: .gitlab-ci.yml (hunk 1 above)
Comment (genalvdom) [reply .gitlab-ci.yml]: It's appear this error
```
ERROR: (gcloud.auth.activate-service-account) unrecognized arguments:
  "service_account",
  "project_id":
  "stashconsulting",
  "private_key_id":
  "2c6725faf2dd841673a79fd61bdf5507774f66c0",
  "private_key":
  "-----BEGIN
```

Comment (genalvdom) [reply .gitlab-ci.yml]: Due to we can run bash in `stashconsulting/terraform-docker:gcloud-latest` image the `prepare` stage was deleted.

**Pull request merged**